code/__DEFINES/skills.dm

```diff
@@ -163,6 +164,12 @@
 #define SKILL_ENDURANCE_SURVIVOR 2
 #define SKILL_ENDURANCE_MAX 5
 
+//cleaning skill, how well you work cleaning equipment
+#define SKILL_CLEANING_NONE 0
+#define SKILL_CLEANING_TRAINED 1
+#define SKILL_CLEANING_MASTER 2
+#define SKILL_CLEANING_MAX 2
+
```

Comment:
this is the literal definition of a bloat skill, it interacts with only one incredibly inconsequential piece of side content. make it a broader skill (and justify its existence) or make it like a character trait or something